samza-core/src/main/java/org/apache/samza/system/StandaloneExecutionEnvironment.java

```diff
@@ -28,7 +28,11 @@
 /**
  * This class implements the {@link ExecutionEnvironment} that runs the applications in standalone environment
  */
-public class StandaloneExecutionEnvironment implements ExecutionEnvironment {
+public class StandaloneExecutionEnvironment extends AbstractExecutionEnvironment {
```

Comment:
Can we rename this to "LocalExecutionEnvironment" ? Standalone is pretty overused and seems like everyone has their own interpretation of it.